tests/test_hypersphere.py

```diff
@@ -396,6 +396,34 @@ def test_mean_and_belongs(self):
         result = self.metric.mean([point_a, point_b, point_c])
         self.assertTrue(self.space.belongs(result))
 
+    def test_random_vonMisesFisher(self):
+        """
+        Check that the maximum likelihood estimates of the mean and
+        concentration parameter are close to the real values for a large
+        sample size.
+        """
+        dim = 2
+        n_points = 1000000
+        sphere = Hypersphere(dim)
+
+        # check mean value for concentrated distribution
+        K = 10
+        points = sphere.random_vonMisesFisher(K, n_points)
+        S = gs.sum(points, axis=0)
+        mu_hat = S/gs.linalg.norm(S)
+        mu = gs.array([0, 0, 1])
+        error = sphere.metric.dist(mu, mu_hat)
+        assert error < 1e-2
```

Comment:
Define the constant: `TOLERANCE = 1e-2` (could we do better? like 1e-6?) at the beginning of the file.

Then:  
```
result = mean_estimate
expected = mean 
self.assertTrue(gs.allclose(result, expected, atol=TOLERANCE))
``` 
in order to use the method of the `unittest.TestCase` class.

Reply:
If we only test the mean value, we can lower the tolerance to 1e-6 by sampling from a much more concentrated distribution (concentration parameter K=100000). See the commit https://github.com/AliceLeBrigant/geomstats/commit/794b49dc7e41927f2a86c43650cdc6d1ff10f330. 

However I could not lower the tolerance below 1e-3 for the estimation of the concentration parameter. What would you recommend ?